bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CallOptionsConfig.java

```diff
@@ -191,7 +253,9 @@ private CallOptionsConfig(Builder builder) {
       readRowsTimeout = longRpcTimeoutMs;
     }
     this.mutateRpcTimeoutMs = mutateTimeout;
-    this.readStreamRpcTimeoutMs = readRowsTimeout;
+    this.mutateRpcAttemptTimeoutMs = builder.mutateRpcAttemptTimeoutMs;
+    this.readStreamRpcTimeoutMs = readRowsTimeout;  // TODO make the names consistent?
```

Comment:
Highlighting this naming inconsistency for @igorbernstein2 

Based on the usages of CallOptionsConfig.getReadStreamRpcTimeoutMs, it seems like the ReadRows name is preferred over ReadStream, so I'd prefer to rename to readRows. LMKWYT